libs/utils/address.go

```diff
@@ -30,11 +30,14 @@ func ValidateAddr(addr string) (string, error) {
 	}
 
 	if ip := net.ParseIP(addr); ip == nil {
-		_, err = net.LookupHost(addr)
+		addrs, err := net.LookupHost(addr)
 		if err != nil {
-			return addr, fmt.Errorf("could not resolve hostname or ip: %w", err)
+			return addr, fmt.Errorf("could not resolve %v, err: %w", addr, err)
```

Comment:
optional, err is not needed, the column separates it well
```suggestion
			return addr, fmt.Errorf("could not resolve %v: %w", addr, err)
```